requirements.txt

```diff
@@ -1,6 +1,7 @@
 # Base requirements for apel
 
-MySQL-python
-python-ldap<3.4.0  # python-ldap-3.4.0 dropped support for Python 2
-iso8601
+mysqlclient==2.1.1
```

Comment:
Why this version? Specifying anything later than 1.4.6 breaks the Travis Python 2 build.

Reply:
Why this version?
-> Acc. to [2.1.1 ](https://pypi.org/project/mysqlclient/2.1.1/). It acknowledges that it works with 3.6 and 3.9 which we tested the codebase and this is the latest package. From 2.2.0 they dropped support for py3.6 officially.

Specifying anything later than 1.4.6 breaks the Travis Python 2 build.
-> Cool. I have added the separate flow for Travis now it should work as expected. Aimed to test codebase with py2 and haven't disturbed the previous libs with version info limitations.